packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -77,10 +134,66 @@
 
   const handleChange = value => {
     const changed = fieldApi.setValue(value)
+    selectedOptions = value.map(val => ({
+      _id: val,
+      [`${primaryDisplay}`]: options.find(option => option._id === val)[
+        `${primaryDisplay}`
+      ],
+    }))
     if (onChange && changed) {
       onChange({ value })
     }
   }
+
+  // Search for rows based on the search string
+  const search = async searchString => {
+    // Reset state if this search is invalid
+    if (!linkedTableId || !searchString) {
+      searchResults = null
+      return
+    }
+
+    // If a filter exists, then do a client side search
+    if (hasFilter) {
+      searchResults = $fetch.rows.filter(option =>
+        option[`${primaryDisplay}`].startsWith(searchString)
```

Comment:
NAB: You don't need a JS template string when your property is already a string - so you could just write this as `option[primaryDisplay]`. I've seen quite a few instances of this so I'll not comment on them all, but just letting you know that you could shorten them all to this syntax.

Reply:
Good catch - updated1